etc/mpv.desktop

```diff
@@ -1,22 +1,41 @@
 [Desktop Entry]
 Type=Application
 Name=mpv Media Player
+Name[de]=Medienwiedergabe »MPV«
 Name[ca]=Reproductor multimèdia mpv
 Name[cs]=mpv přehrávač
 Name[da]=mpv-medieafspiller
 Name[fr]=Lecteur multimédia mpv
+Name[fr_BE]=MPV Lecteur multimédia
+Name[hi]=एमपीवी मीडिया प्लेयर
+Name[it]=Lettore multimediale MPV
 Name[ja]=mpv メディアプレイヤー
+Name[or]=MPV ମିଡିଆ ପ୍ଲେୟାର |
+Name[pt]=Leitor de multimédia MPV
+Name[pt_BR]=Reprodutor de Multimídia MPV
 Name[pl]=Odtwarzacz mpv
 Name[ru]=Проигрыватель mpv
+Name[sv]=MPV mediaspelare
 Name[tr]=mpv Ortam Oynatıcı
 Name[zh_CN]=mpv 媒体播放器
 Name[zh_TW]=mpv 媒體播放器
 GenericName=Multimedia player
 GenericName[cs]=Multimediální přehrávač
 GenericName[da]=Multimedieafspiller
+GenericName[de]=Multimedia-Wiedergabeprogramm
 GenericName[fr]=Lecteur multimédia
+GenericName[fr_BE]=Lecteur multimédia
+GenericName[hi]=मल्टीमीडिया प्लेयर
+GenericName[hu]=Multimédia lejátszó
+GenericName[it]=Lettore multimediale
 GenericName[ja]=マルチメディアプレイヤー
+GenericName[or]=ମଲ୍ଟିମିଡ଼ିଆ ପ୍ଲେୟାର
+GenericName[pt]=Leitor de multimédia
+GenericName[pt_BR]=Reprodutor de Multimídia MPV
```

Comment:
sloppy copy-pasting

Reply:
The menus with the program names in lower case are too ugly. This looks like some sort of inferiority complex.

- - - - -

Os menus com os nomes de programas em letras minúculas ficam demasiadamente feios. Isso se parece com algum tipo de complexo de inferioridade.